governance.qmd

```diff
@@ -0,0 +1,96 @@
+---
+title: "R-Multiverse Governance"
```

Comment:
should this live under github.com/r-multiverse in `.github/GOVERNANCE.md`, and be read from the qmd?

Reply:
I don't have a specific view on this, except that we shouldn't have a situation where the two can get out of sync.